scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryTypes.scala

```diff
@@ -64,18 +64,56 @@ final case class Query(underlying: String) extends Source {
   def latest(): Query = latest(BigQuery.defaultInstance())
 }
 
+/**
+ * A wrapper type [[Table]] which wraps a table spec String and a table [[GTableReference]]
```

Comment:
```suggestion
 * [[Table]] abstracts the multiple ways of referencing Bigquery tables. Tables can be referenced by a table spec `String` or by a table reference [[GTableReference]]. 
```

Reply:
@liufuyang I think this suits it better.

Reply:
Indeed. I had the concept but not exactly sure how to phrase it. Thank you very much for the suggestion 👍